ls1intum/Artemis
# `Development`: Remove application links from Atlassian documentation

### Checklist
#### General

### Motivation and Context


After #6682 we don't need the application links anymore, this PR removes their setup from the documentation.

### Description


### Steps for Testing
Confirm that all mentions of application links are removed and the script `atlassian-setup.sh` still works as expected.

https://artemis-platform--7244.org.readthedocs.build/en/7244/dev/setup.html#bamboo-bitbucket-and-jira-setup

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test
